biome.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Every time a new tool uses JSON as its configuration for humans, an angel is cast down... :(